src/markdown-pages/build-apps/build-visualization.mdx

```diff
@@ -56,6 +56,8 @@ Start with a new Nerdpack. If you just installed the New Relic One CLI, then you
 nr1 create --type nerdpack --name my-awesome-nerdpack
 ```
 
+_Note: If you only want to build a visualization in your Nerdpack, you can remove the `nerdlets/` and `launchers/` folders from the nerdpack_
```

Comment:
I'd recommend using a `<Callout variant="tip">` here to highlight this note.

Reply:
Thanks! Updated to use the Callout component :)